scripts/keys/README.md

```diff
@@ -0,0 +1,47 @@
+# Release verification keys
+
+This directory contains all keys that are currently signing `pool` releases.
+
+The name of the file must match exactly the suffix that user is going to use
+when signing a release. For example, if the key is called `signer.asc` then 
+that user must upload a signature file called `manifest-signer-v0.xx.yy-beta.sig`,
+otherwise the verification will fail. You can check [lnd's release documentation](https://github.com/lightningnetwork/lnd/blob/master/docs/release.md#signing-an-existing-manifest-file) 
+for details on how to create the signature.
+
+In addition to adding the key file here as a .asc file the 
+`scripts/verify-install.sh` file must also be updated with the key ID and the
+reference to the key file.
+
+# Verifying the Release
+
+In order to verify the release, you'll need to have gpg or gpg2 installed on 
+your system. Once you've obtained a copy (and hopefully verified that as well),
+you'll first need to import the keys that have signed this release if you 
+haven't done so already:
+
+```
+curl https://raw.githubusercontent.com/lightningnetwork/lnd/master/scripts/keys/guggero.asc | gpg --import
```

Comment:
nit: change this to the Pool repo? Or use a key server as shown [here](https://github.com/lightninglabs/faraday/releases/tag/v0.2.8-alpha)?

Reply:
Or, thinking about it, should we instead use the keys in the lnd repo for verification in the first place? Since if you trust the lnd signers, you probably also trust them to sign for Pool?
Meaning we wouldn't commit any keys to this repo in the first place (and update the `verify-install.sh` script accordingly)?